packages/js/test-env/src/index.ts

```diff
@@ -27,6 +21,12 @@ export const providers = {
   ethereum: "http://localhost:8545",
 };
 
+export const embeddedWrappers = {
+  ens: `wrap://fs/${path.join(__dirname, "wrappers", "ens")}`,
+  uts46: `wrap://fs/${path.join(__dirname, "wrappers", "uts46")}`,
+  sha3: `wrap://fs/${path.join(__dirname, "wrappers", "sha3")}`,
```

Comment:
Why do we need to embed uts46 & sha3?

Reply:
I think that the reason is that ENS wrapper needs those to work

Reply:
tests sometimes fail to resolve the URIs and fetch the wrappers. It happens enough to make it hard to get all of the CI tests to pass at the same time.